datasets/codexglue_concode/codexglue_concode.py

```diff
@@ -0,0 +1,92 @@
+"""concode - code2text pairs dataset"""
+import datasets
+import json
+import logging
+
+_CITATION = """\
+@article{iyer2018mapping,
+  title={Mapping language to code in programmatic context},
+  author={Iyer, Srinivasan and Konstas, Ioannis and Cheung, Alvin and Zettlemoyer, Luke},
+  journal={arXiv preprint arXiv:1808.09588},
+  year=2018,
+}
+"""
+_URL = "https://raw.githubusercontent.com/microsoft/CodeXGLUE/main/Text-Code/text-to-code/dataset/concode/"
+_URLS = {
+    "train": _URL + "train.json",
+    "dev": _URL + "dev.json",
+    "test": _URL + "test.json",
+}
+_DESCRIPTION = """Mapping Language to Code in a Programmatic Context"""
```

Comment:
Could you add more details ? Where does the dataset come from ? How big is it etc.